src/app/kvm/components/KVMHeaderForms/AddLxd/SelectProjectForm/SelectProjectForm.test.tsx

```diff
@@ -86,35 +79,25 @@ describe("SelectProjectForm", () => {
     state.pod.projects = {
       "192.168.1.1": [project],
     };
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/kvm/add", key: "testKey" }]}
-        >
-          <CompatRouter>
-            <SelectProjectForm
-              clearSidePanelContent={jest.fn()}
-              newPodValues={newPodValues}
-              setStep={jest.fn()}
-              setSubmissionErrors={jest.fn()}
-            />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithBrowserRouter(
+      <SelectProjectForm
+        clearSidePanelContent={jest.fn()}
+        newPodValues={newPodValues}
+        setStep={jest.fn()}
+        setSubmissionErrors={jest.fn()}
+      />,
+      { route: "/kvm/add", state }
     );
 
-    wrapper.find("input[name='newProject']").simulate("change", {
-      target: { name: "newProject", value: "foo" },
+    const nameInput = screen.getByRole("textbox", {
+      name: /New project name/i,
     });
-    wrapper.find("input[name='newProject']").simulate("blur");
-    await waitForComponentToPaint(wrapper);
-
+    await userEvent.clear(nameInput);
+    await userEvent.type(nameInput, "foo");
+    fireEvent.blur(nameInput);
```

Comment:
```suggestion
    await userEvent.type(nameInput, "foo");
    await userEvent.tab();
```

Reply:
I tried that before defaulting to fireEvent but it didn't seem to work